apps/verticalhome/js/clock.js

```diff
@@ -0,0 +1,86 @@
+/**
+  Copyright 2012, Mozilla Foundation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+'use strict';
```

Comment:
I think it is more useful to put the 'use strict' inside the '(function(exports) {'.  When this code ships, all the files will be concatenated. So most 'use strict' directives like this one will not be the first expression in the file and will have no effect.  If you put them as the first expression inside a function then they will always have an effect.